client/src/Components/InterviewHeader.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { useStyles } from '../themes/theme';
+import Grid from '@material-ui/core/Grid';
+import LanguageSelectMenu from './LanguageSelectMenu'
+import { ExitInterviewButton } from '../components/Buttons'
+import { useHistory } from 'react-router-dom'
+
+const InterviewHeader = (props) => {
```

Comment:
I generally like to use the destructuring assignment for props, so you don't have to write props.setLanguage, etc. 

So it would look like:
`const InterviewHeader = ({setLanguage}) => {`